app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for entry in people:
+        person = Person(entry["name"], entry["age"])
+        persons.append(person)
```

Comment:
This violates checklist item 3: "When creating a list of instances from a collection (such as a list of dictionaries), it is recommended to use list comprehension rather than traditional `for` loops." The code uses a traditional loop to build `persons` (lines 11-14). Prefer a list comprehension to construct the Person instances so the code follows the checklist guidance.